src/charm.py

```diff
@@ -55,11 +55,11 @@ def _on_website_relation_joined(self, event):
             self.unit.status = BlockedStatus('machine does not appear to be a controller')
             return
 
-        ingress_address = hookenv.ingress_address(event.relation.id, hookenv.local_unit())
+        address = hookenv.unit_private_ip()
```

Comment:
Just confirming - as discussed IRL, we want to avoid the use of private address (which is deprecated) and instead use network-info to get the binding address value. If that doesn't work, we'll need to look into why.